src/test/debuginfo/vec-slices.rs

```diff
@@ -53,6 +53,11 @@
 // gdb-command:print *((int64_t[2]*)('vec_slices::MUT_VECT_SLICE'.data_ptr))
 // gdb-check:$15 = {64, 65}
 
+//gdb-command:print mut_slice.length
```

Comment:
Well, that's what we've been trying to fix here 😄 `print mut_slice` should show an array, just like `print slice`